src/js/components/Search/SearchResults.js

```diff
@@ -15,13 +27,47 @@ const SearchResults = () => {
 
   return (
     <div style={wrapperStyle}>
-      <Card style={{ borderRadius: '10px' }}>
-        <Statistic
-          title="Count"
-          value={isValid ? (status === 'count' ? count : message) : '----'}
-          valueStyle={{ color: '#1890ff' }}
-          prefix={<TeamOutlined />}
-        />
+      <Card style={{ borderRadius: '10px', padding: '10px 33px', width: '900px' }}>
+        <Row gutter={16}>
+          <Col span={6}>
+            <Space direction="vertical" size="middle" style={{ display: 'flex' }}>
+              <Statistic
+                title="Individual Count"
+                value={isValid && status === 'count' ? (status === 'count' ? count : message) : '----'}
+                valueStyle={{ color: '#1890ff' }}
+                prefix={<TeamOutlined />}
+              />
+              <Statistic
```

Comment:
I think we considered this component and the following one to be displayed only when there is enough data?

Reply:
the value wouldn't be returned by katsu, so it cannot really show anything